README.md

```diff
@@ -90,9 +89,9 @@ $ npm run build-lib
 
 If you want to add more components, services and so on to a module of the library, you can do it with:
 
-`$ ng generate component [path/in/your/module/][name-of-component] --project knora-ui --styleext scss --prefix kui`
+`$ ng generate component [path/in/your/module/][name-of-component] --project @knora/ui --styleext scss --prefix kui`
 
-Before testing the new component inside of the demo app, you have to rebuild after each change: `ng build knora-ui`.
+Before testing the new component inside of the demo app, you have to rebuild after each change: `ng build @knora/ui`.
```

Comment:
I think we should always use the npm scripts: `npm run build-lib` and `npm run build-app`. This is not only for convenience, but also because then the locally installed version from `node_modules` is used. The versions are then controlled by `package.json`.

If we use global versions then this could lead to different behaviour depending on the version of the globally installed package.

Reply:
agree

Reply:
This should be part of other PR and user story e.g. https://dasch.myjetbrains.com/youtrack/issue/DSP-199

Reply:
It's on the list in https://dasch.myjetbrains.com/youtrack/issue/DSP-185